### ISSUE TITLE
Dockerisation of airquality-appabz throws exceptions

### ISSUE THREAD

**Issue by bertolla** [noi-techpark/bdp-commons#223]
I adapted the whole configuration to Docker (see pipelines branch) and ones deployed the app starts correctly. When the elaboration is triggered, files are being downloaded in tmp folders, but the unzipping fails(in this case mapping.zip). 
@Piiit Is this a known issue which needed manual intervention?
```
app_1  | 2020-12-22 15:00:01,445 [Scheduler-1] FEEDBACK it.bz.idm.bdp.airquality.DataCom - Fetching uploads/mapping.zip
app_1  | 2020-12-22 15:00:01,466 [Scheduler-1] ERROR org.springframework.scheduling.support.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
app_1  | java.io.FileNotFoundException: /tmp/bdp-airquality/meta/mapping/errors.csv (Not a directory)
app_1  | 	at java.io.FileOutputStream.open0(Native Method) ~[?:1.8.0_275]
app_1  | 	at java.io.FileOutputStream.open(FileOutputStream.java:270) ~[?:1.8.0_275]
app_1  | 	at java.io.FileOutputStream.<init>(FileOutputStream.java:213) ~[?:1.8.0_275]
app_1  | 	at java.io.FileOutputStream.<init>(FileOutputStream.java:162) ~[?:1.8.0_275]
app_1  | 	at it.bz.idm.bdp.airquality.utils.FileTools.unZip(FileTools.java:83) ~[classes/:?]
app_1  | 	at it.bz.idm.bdp.airquality.DataCom.fetchMetaDataFromFTP(DataCom.java:72) ~[classes/:?]
app_1  | 	at it.bz.idm.bdp.airquality.JobScheduler.pushDataToCollector(JobScheduler.java:45) ~[classes/:?]
app_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_275]
app_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_275]
app_1  | 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_275]
app_1  | 	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_275]
app_1  | 	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:84) ~[spring-context-5.2.11.RELEASE.jar:5.2.11.RELEASE]
app_1  | 	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) [spring-context-5.2.11.RELEASE.jar:5.2.11.RELEASE]
app_1  | 	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:93) [spring-context-5.2.11.RELEASE.jar:5.2.11.RELEASE]
app_1  | 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_275]
app_1  | 	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_275]
app_1  | 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_275]
app_1  | 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_275]
app_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_275]
app_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_275]
app_1  | 	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
```

**Comment by Piiit** [comment]
@bertolla Never seen this issue before, it seems that inside docker the files/paths may be different... Need to check before I can tell you more. Until when do you need it?

**Comment by bertolla** [comment]
In Docker the Filestructure is being created. The unzipping fails

**Comment by bertolla** [comment]
I fixed this issue in this commit a9b063874ba7f73eb903001fd47dccb0f4092af1 way back but forgot to close the issue.
